rustls/src/vecbuf.rs

```diff
@@ -9,10 +9,11 @@ use std::io::Read;
 #[cfg(feature = "std")]
 use crate::msgs::message::OutboundChunks;
 
-/// This is a byte buffer that is built from a vector
-/// of byte vectors.  This avoids extra copies when
-/// appending a new byte vector, at the expense of
-/// more complexity when reading out.
+/// This is a byte buffer that is built from a deque
+/// of byte vectors.
```

Comment:
Nit: unfold this? I think we're allowed to go until column 100 which I'm pretty sure this isn't hitting.